### PR TITLE
Create PAC pipelinerun templates for console to access

### PR DESCRIPTION
**Pull request by sm43**
this fetches the pr templates while fetching release yaml and keep in ko data
later it creates configmaps with pipelinerun templates and add in installer set
console will use this templates.
also adds rbac to access templates for non admin users

Signed-off-by: Shivam Mukhade <smukhade@redhat.com>

# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/operator/blob/master/CONTRIBUTING.md) for more details._

# Release Notes

### CODE REVIEW

File: hack/fetch-releases.sh [hunk 1]
```diff
@@ -122,6 +122,27 @@ release_yaml_pac() {
 
     echo "Info: Added $comp/$fileName:$version release yaml !!"
     echo ""
+
+    runtime=( go java nodejs python )
+    for run in "${runtime[@]}"
+    do
+    	echo "fetching PipelineRun template for runtime: $run"
+
+    	source="https://raw.githubusercontent.com/sm43/pipelines-as-code/pac-pr-templates/pkg/cmd/tknpac/generate/templates/${run}-template.yaml"
```
Comment (vdemeester) [inline hack/fetch-releases.sh:131]: This should probably point to `openshift-pipelines/pipelines-as-code` ?

**Review by vdemeester — COMMENTED** [review]
SGTM, one question though.

File: pkg/reconciler/openshift/tektonaddon/pipelinesascode.go [hunk 2]
```diff
@@ -65,12 +82,18 @@ func (r *Reconciler) EnsurePipelinesAsCode(ctx context.Context, ta *v1alpha1.Tek
...
 
+	// core manifest
 	koDataDir := os.Getenv(common.KoEnvKey)
 	pacLocation := filepath.Join(koDataDir, "tekton-addon", "pipelines-as-code")
 	if err := common.AppendManifest(&pacManifest, pacLocation); err != nil {
 		return err
 	}
 
+	if err := fetchPRTemplates(&pacManifest); err != nil {
+		fmt.Println("failed here ...")
```
Comment (piyush-garg) [inline pkg/reconciler/openshift/tektonaddon/pipelinesascode.go:93]: this needs to be removed i think :D

File: hack/fetch-releases.sh (hunk 1 above)
Comment (sm43) [reply hack/fetch-releases.sh:131]: yeah the PAC pr was not merged 😅 then!

**Review by vdemeester — APPROVED** [review]

**Comment by piyush-garg** [comment]
we missed the license header in files

**Pull request merged**